ssl/src/main/java/io/crate/protocols/ssl/SslConfigSettings.java

```diff
@@ -50,6 +51,9 @@ private SslConfigSettings() {}
     public static final CrateSetting<Boolean> SSL_PSQL_ENABLED = CrateSetting.of(
         Setting.boolSetting(SSL_PSQL_ENABLED_SETTING_NAME, false, Setting.Property.NodeScope),
         DataTypes.BOOLEAN);
+    public static final CrateSetting<Boolean> SSL_MQTT_ENABLED = CrateSetting.of(
```

Comment:
I think this setting definition should belong in the mqtt module, not inside ssl.

If this conflicts with how `isSslEnabled` is used we've to rethink that. Imho nothing that is purely mqtt related should leak into any other module.